Club Penguin Rewritten/dist/metadata.json

```diff
@@ -8,17 +8,17 @@
     "en": "A new free virtual world based on Disney's Club Penguin with weeky updates. Waddle around and meet new friends!"
   },
   "url": "play.cprewritten.net",
-  "version": "3.0.0",
+  "version": "3.0.2",

```

Comment:
Change the version to `3.0.1`

Reply:
I was told by forzaum to do 3.0.2 because the revert was 3.0.1

Reply:
#1461

Reply:
Resolve the conflicting files.

Reply:
what conflicting files? what exactly would resolving that look like?

Reply:
@BartBoys3 
![image](https://user-images.githubusercontent.com/38713645/80792808-8bf5bd00-8be9-11ea-92c8-b66f2c2820e0.png)

Reply:
I see the button, but when I press it, I don't see what to change.

Reply:
Conflicts have been resolved